src/server/services/bounty.service.ts

```diff
@@ -404,7 +404,11 @@ export const getImagesForBounties = async ({ bountyIds }: { bountyIds: number[]
   });
 
   const groupedImages = groupBy(
-    connections.map(({ entityId, image }) => ({ ...image, entityId })),
+    connections.map(({ entityId, image }) => ({
+      ...image,
+      tags: image.tags.map((t) => ({ id: t.tag.id, name: t.tag.name })),
```

Comment:
Just wanted to leave a comment 😛 
```suggestion
      tags: image.tags.map(({ tag }) => tag),
```

Reply:
Want to avoid extra-tag attributes so that typescript complaints less in the future <3